Kyung removed gtk requirement

Jothle12 opened the pull request:
Pygtk was not in use, but was required for startup before. Now it isn't.

kyung01 committed:
Now runs on window as well

kyung01 committed:
New version

lcb931023 committed:
Re-enabling event

Commit:
Kyung changed Main.py to not require pygtk

Nolski commented on Main.py:
```diff
@@ -3,7 +3,10 @@
 import os
 import json
-from gi.repository import Gtk
+#from gi.repository import Gtk
```
Everything looks good. Is there any reason why this one is commented out and not just removed?

kyung01 committed:
GTK requirement gone, input latency resolved.

Input latency is now no more.
TODO
0) pump up frame rate! Now it's currently running at 2.5 frames per
second

lcb931023 committed:
Timer bonus added.

lcb931023 committed:
disable check answer when win screen is up

Commit:
Added exit button to "how to play menu"
-fixes #5

Merge branch 'master' of https://github.com/Jothle12/xo-fractions-game

Conflicts:
	Main.py

nekocoder committed:
Added a 1/4th second delay to the button click events in the "How to Play" screen

fixes #5

lcb931023 committed:
Changed activity logo to a cute lil rocket.

lcb931023 committed:
Merge branch 'master' of https://github.com/kyung01/xo-fractions-game

lcb931023 committed:
0f6f696 fixed #9. Now please close yourself, issue!

kyung01 committed:
Optimizing, trying to fuck

kyung01 committed:
SceneBasicDone

kyung01 committed:
Scenegame

kyung01 committed:
SceneGame

kyung01 committed:
Done

lcb931023 committed:
Added scoreboard summary.

Closes #8

kyung01 committed:
Resolution side set to small

kyung01 committed:
helperRaiseEventHDR to helperRaiseEvent

kyung01 committed:
SceneHelpButton Input fixed

lcb931023 committed:
Added Highscores; Disabled score saving.

Fixes #13

lcb931023 committed:
minor UI rearrangement

JordanLehman committed:
Adding Question to Everything

kyung01 committed:
Question : Organizing/Need more organization.

kyung01 committed:
SceneGame : ObserverPattern polished.

kyung01 committed:
Rebuilding from the ground up, almost.

kyung01 committed:
TRIVIAL Recording progress

lcb931023 committed:
Reskin assets for Main Menu

kyung01 committed:
TRIVIAL Recording Progoress

JordanLehman committed:
Removed printlines from testing.

Commit:
Merge branch 'Optimization' of https://github.com/Kyung01/xo-fractions-game into Optimization

Conflicts:
	Question.py
	SceneGame.py

Commit:
small changes

Commit:
Fitting my old code to recent changes.

Commit:
Added more get functions to work with current format of load level.

Commit:
Small Change

kyung01 committed:
Drastic change in "level loading method"

kyung01 committed:
ScreenGame : Cleaning deprecated codes.

kyung01 committed:
TRIVIAL ScreenWin being worked on

kyung01 committed:
Level Advancing Win State implemented

kyung01 committed:
Changbai's images imported from main branch.

kyung01 committed:
MainScreen ART NOW IN!

lcb931023 committed:
Periodic progress of game screen; fuel container new design

kyung01 committed:
Bars moving around now

kyung01 committed:
IcnFuel Animated

kyung01 committed:
IcnFuel Animation Wave

kyung01 committed:
TRIVIAL working on rocket

kyung01 committed:
TRIVIAL name change IcnOil -> IcnFuel

kyung01 committed:
IcnRocket displaying chosen amount of fuel now

kyung01 committed:
Text boxs being displayed.

kyung01 committed:
TRIVIAL ScreenGame

kyung01 committed:
TRIVIAL

kyung01 committed:
TRIVIAL adding more interface

kyung01 committed:
BottomFooter Added

kyung01 committed:
Resets question when incorrect answer

lcb931023 committed:
Art Assets for gameplay WOO

lcb931023 committed:
Merge branch 'Optimization' of https://github.com/kyung01/xo-fractions-game into Optimization

kyung01 committed:
WinScreen skips after a few sec

lcb931023 committed:
change rocket fuel box color thing

lcb931023 committed:
main-bg-nostar

lcb931023 committed:
SHOOTING STARZ THATS RIGHT

kyung01 committed:
Reskin fuel/rocket

kyung01 committed:
TRIVIAL bugs

kyung01 committed:
TRIVIAL cleaning

kyung01 committed:
Hardcoded lines & text

kyung01 committed:
TRIVIAL ScreenMenu being worked on

kyung01 committed:
BasicParticle Done : PERFORMANCE?!NO?

kyung01 committed:
Particle has random property.

However the particles are not "shiny" enough.
We need something more than just these falling stars....

kyung01 committed:
Merge branch 'Optimization' of https://github.com/kyung01/xo-fractions-game into Optimization

Conflicts:
	SceneMenu.py

kyung01 committed:
MergeConflict resolved

Conflicts:
	SceneMenu.py

kyung01 committed:
ShootingStart Effect : Screen distortion.

kyung01 committed:
Optimization improving rendering

kyung01 committed:
TRIVIAL new type of particle being worked

kyung01 committed:
TRIVIAL particle for rocket.

JordanLehman committed:
changed Question to QuestionMaker, Changed format, imported in SceneGame

kyung01 committed:
TRIVIAL merge conflict

kyung01 committed:
TRIVIAL bug

kyung01 committed:
Optimization

lcb931023 committed:
Finalized gameplay art, as well as back button

kyung01 committed:
Optimization

kyung01 committed:
TRIVIAL glitch

lcb931023 committed:
implement finalized art asset

kyung01 committed:
Changbai requested feature : text color

lcb931023 committed:
delete redundant asset

lcb931023 committed:
final asset integration

lcb931023 committed:
TRIVIAL bug

kyung01 committed:
TRIVIAL merge

kyung01 committed:
Revert "TRIVIAL merge"

This reverts commit d6ea33dd563ca4e54039227ec1fee982fc3959b6.

kyung01 committed:
Revert "Revert "TRIVIAL merge""

This reverts commit 57b91a110175218649ab22ca347e317f18a98627.

lcb931023 committed:
Getting ready for the merge hell... I mean caping the texts

lcb931023 committed:
merging conflict

kyung01 committed:
FuelBars smooth ease in/out animation

kyung01 committed:
SoundEffectHadBeenAdded

kyung01 committed:
TextAnimated Now

Commit:
Added a new sound file to the sound directory (IT HAS NOT BEEN ADDED IN THE CODE)

kyung01 committed:
Renamed Question -> QuestionMaker

Commit:
Merge branch 'Optimization' of https://github.com/kyung01/xo-fractions-game into Optimization

kyung01 committed:
DynamicLevel generated now

kyung01 committed:
QuestionMaker error / removed for now

SCENE_BASIC_ENTER
LOADING NEW QUESTION
LOADING NEW QUESTION
EVENT_CLICK
updating rocket 0.5
EVENT_CLICK
Next frame now
LOADING NEW QUESTION
LOADING NEW QUESTION
EVENT_CLICK
updating rocket 0.5
EVENT_CLICK
Next frame now
Exception in thread Thread-1:
Traceback (most recent call last):
File "C:\Python27\lib\threading.py", line 551, in __bootstrap_inner
self.run()
File "C:\Python27\lib\threading.py", line 504, in run
self.__target(*self.__args, **self.__kwargs)
File "C:\Users\Kyunghwan\Documents\GitHub\xo-fractions-game\Main.py",
line 137
, in loopRender
self.dicScenes[self.myState].renderUpdate(self.clock.get_time() * .001)
File
"C:\Users\Kyunghwan\Documents\GitHub\xo-fractions-game\SceneWin.py",
line
71, in renderUpdate
s.helperRaiseEvent(s.EVENT_FINISHED)
File
"C:\Users\Kyunghwan\Documents\GitHub\xo-fractions-game\SceneBasic.py",
li
ne 11, in helperRaiseEvent
for e in events: e();
File "C:\Users\Kyunghwan\Documents\GitHub\xo-fractions-game\Main.py",
line 71,
in EVENTHDR_SCENE_CONTINUE_GAME
self.changeState(self.STATE_GAME)
File "C:\Users\Kyunghwan\Documents\GitHub\xo-fractions-game\Main.py",
line 150
, in changeState
self.dicScenes[stateNew].EVENT_SCENE_START()
File
"C:\Users\Kyunghwan\Documents\GitHub\xo-fractions-game\SceneGame.py",
lin
e 309, in EVENT_SCENE_START
self.initLevel()
File
"C:\Users\Kyunghwan\Documents\GitHub\xo-fractions-game\SceneGame.py",
lin
e 192, in initLevel
self.questionMaker.makeNextQuestion(self.questionLevel)
File
"C:\Users\Kyunghwan\Documents\GitHub\xo-fractions-game\QuestionMaker.py",
line 61, in makeNextQuestion
goalMod = denom % 5
UnboundLocalError: local variable 'denom' referenced before assignment

SceneWin Click

kyung01 committed:
QuestionMaker Glitch

kyung01 committed:
ScreenWin hooked up art

JordanLehman committed:
Little late, but I have incremental difficulty.

lcb931023 committed:
Mouse Cursor

lcb931023 committed:
light strip assets

lcb931023 committed:
TRIVIAL change win screen bg

kyung01 committed:
Sound from eric's branch

kyung01 committed:
MouseCursor Added

kyung01 committed:
Levels added

kyung01 committed:
backgroundmusic decided

kyung01 committed:
SOUND : rocektVibrating

kyung01 committed:
SOUND : too much fuel

kyung01 committed:
GLITCH : added try/catch loop "insurance"

kyung01 committed:
SOUND : gameStart/gameExit

kyung01 committed:
GLITCH : sound sampling sound,

kyung01 committed:
GLITCH

kyung01 committed:
GLITCH

kyung01 committed:
GLITCH

kyung01 committed:
GLITCH

kyung01 committed:
GLITCH

kyung01 committed:
FrameRate Fixed

kyung01 committed:
GLITCH Sound speed wtf

kyung01 committed:
GLITCH sound fuck

kyung01 committed:
GLITCH main reverted

kyung01 committed:
Revert "GLITCH main reverted"

This reverts commit 0f5721913b7a05e007eeb3143b463b350c6435d5.

kyung01 committed:
Glitch

kyung01 committed:
Glitch

kyung01 committed:
omg please

kyung01 committed:
please please please

kyung01 committed:
Please one more time please

kyung01 committed:
;_; why not working

kyung01 committed:
weqwqekweqjweqlwejq

kyung01 committed:
FIXEDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD

kyung01 committed:
That last of last edit

lcb931023 committed:
spaces between encouragements

lcb931023 committed:
switch back to 1200 900 screen res

lcb931023 committed:
Fullscreen now has no border on XOPC

lcb931023 committed:
Resolve Conflict

lcb931023 committed:
Merging branch Optimization

lcb931023 commented:
Lads and gents, I present to you......

Fractionauts 2.0!
![cover-photo](https://cloud.githubusercontent.com/assets/3478059/2980048/a9df5a18-dbde-11e3-829b-6f16f95dc3c8.png)

Or shall we say, Fractionauts 4/2? XD

(Sorry about the obviously non-related name. We wrote a pull request ages ago, and it seems Github doesn't feel like allowing us more pull requests till this one is merged.)

LanceALaughlin commented:
@lcb931023 You should look into gitflow to mitigate the issue with Github only allowing one pull request.

lcb931023 committed:
Sugarfy the icon

lcb931023 commented:
Will do, will do.
Can we merge this one? The quicker the better, to avoid more stacked commits.

LanceALaughlin commented:
Merged!
